### PR TITLE
[Remove Stream/Optional usage] LPS-174106 remove_stream_and_optional_in_roles_admin_web

### PR DESCRIPTION
**Pull request by jeffwu0724**
Hi Tina,

This is for ticket https://issues.liferay.com/browse/LPS-174106.
Thanks for checking!

### CODE REVIEW

**Comment by catarineaguiar** [comment]
ci:test:roles

File: modules/apps/roles/roles-admin-web/src/main/java/com/liferay/roles/admin/web/internal/group/type/contributor/util/GroupTypeContributorUtil.java [hunk 1]
```diff
@@ -36,19 +34,20 @@
...
+
+			if (!groupTypeContributor.isEnabled()) {
+				continue;
+			}
+
+			classNameIds.add(
+				PortalUtil.getClassNameId(groupTypeContributor.getClassName()));
+		}
+
+		return ListUtil.toLongArray(classNameIds, Long::valueOf);
```
Comment (tinatian) [inline modules/apps/roles/roles-admin-web/src/main/java/com/liferay/roles/admin/web/internal/group/type/contributor/util/GroupTypeContributorUtil.java:50]: @jeffwu0724 Please do not use ListUtil here, use ArrayList.toArray. The ArrayList version do a array copy without iterating.

**Comment by tinatian** [comment]
@jeffwu0724 Please see my comment and resend.

Tina

**Comment by tinatian** [comment]
ci:forward

**Pull request closed without merging**